pom.xml

```diff
@@ -20,7 +20,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>ch.cern.eam</groupId>
     <artifactId>eam-wshub-core</artifactId>
-    <version>11.5-6-SNAPSHOT</version>
+    <version>11.5-6</version>
```

Comment:
The release plugin is the one that sets the version on the pom (everything else is just a snapshot of the next version)